saleor/graphql/translations/mutations.py

```diff
@@ -106,10 +106,16 @@ def perform_mutation(cls, _root, info, **data):
 
         product_pk = cls.get_global_id_or_error(data["id"], only_type=Product)
         product = product_models.Product.objects.get(pk=product_pk)
-        product.translations.update_or_create(
+        translation, created = product.translations.update_or_create(
             language_code=data["language_code"], defaults=data["input"]
         )
         product = ChannelContext(node=product, channel_slug=None)
+
+        if created:
+            info.context.plugins.translation_created(translation)
+        else:
+            info.context.plugins.translation_updated(translation)
```

Comment:
Maybe a bit unrelated to this PR, but this mutation should probably use transaction and these plugin calls should be made in transaction.on_commit. I don't know why it's not used already, as it is done in `ProductVariantTranslate`; maybe it's a good occasion to unify it.